README.md

```diff
@@ -43,42 +43,65 @@ Autocommands in Vim allow you to execute commands automatically in response to s
 
 1. Highlight specific words in files with a `.tid` extension:
    ```
-   autocmd BufRead,BufNewFile *.tid syntax match InnovationGroup "\<[Ii]nnovat\w*\>"
+   if has("autocmd")
```

Comment:
Leave the README.md unchanged, since the section should only focus on the autocmd. The conditional is not important here.